src/commands/install.yml

```diff
@@ -20,15 +20,14 @@ steps:
       command: |
 
         # If the version is not explicitly specified then use the existing one if possible.
-        if [ "<< parameters.version >>" = "latest" ]  && command -v secrethub >/dev/null 2>&1; then
+        if [ "<< parameters.version >>" = "latest" ] && command -v secrethub >/dev/null 2>&1; then
             exit 0
         fi
 
         # Get the latest version.
         VERSION="<< parameters.version >>"
         if [ "$VERSION" = "latest" ]; then
-            VERSION=$(curl --silent --show-error --location --fail --retry 3 \
```

Comment:
What's `$4` in this case?

Reply:
It's parsing the json: `"tag_name": "v0.35.0"`

Reply:
This is the one-liner we use in `get.secrethub.io` as well.